homeassistant/components/traccar_server/__init__.py

```diff
@@ -54,6 +64,16 @@ async def async_setup_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
 
     await hass.config_entries.async_forward_entry_setups(entry, PLATFORMS)
     entry.async_on_unload(entry.add_update_listener(async_reload_entry))
+    if entry.options.get(CONF_EVENTS):
+        entry.async_on_unload(
+            async_track_time_interval(
```

Comment:
If we're going to add event entities for the fetched events, I'd probably use a separate coordinator for that.

Reply:
Events are a part of the websocket implementation, but it requires user configuration, so once moved to it it will be breaking